pkg/controller/gslb/status.go

```diff
@@ -31,6 +31,8 @@ func (r *ReconcileGslb) updateGslbStatus(gslb *ohmyglbv1beta1.Gslb) error {
 		return err
 	}
 
+	gslb.Status.GeoTag = os.Getenv("CLUSTER_GEO_TAG")
```

Comment:
I'll create separate issue for moving the rest of configuration into depresolver.
the goal is something like this: `gslb.Status.GeoTag = r.config.ClusterGeoTag`

Reply:
yeah, great stuff